### PR TITLE
Remove wcoss_dell_p3 from workflow

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES: 
As WCOSS1 goes away on 7/15/22, `wcoss_dell_p3` is removed from the regional workflow.

## TESTS CONDUCTED: 
- WE2E tests on Hera:
MET_verification
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp_regional
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson_mynn_lam3km
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_RRFS_v1beta

### CODE REVIEW

File: modulefiles/module-setup.csh.inc [hunk 1]
```diff
@@ -13,34 +13,12 @@ else if ( { test -d /scratch3 } ) then
...
-    unset _LMFILES_
-    unset LOADEDMODULES
-    module use /opt/modulefiles
-    module use /opt/cray/ari/modulefiles
-    module use /opt/cray/craype/default/alt-modulefiles
-    module use /opt/cray/alt-modulefiles
-    module use /gpfs/hps/nco/ops/nwprod/modulefiles
-    module use /gpfs/hps/nco/ops/nwprod/lib/modulefiles
-    module use /usrx/local/prod/modulefiles
 else if ( { test -d /dcom -a -d /hwrf } ) then
```
Comment (BenjaminBlake-NOAA) [inline modulefiles/module-setup.csh.inc]: This block is from the version of WCOSS that is even older than Mars/Venus/Luna/Surge (Tide and Gyre), so I suggest removing it.

File: modulefiles/module-setup.sh.inc [hunk 2]
```diff
@@ -28,35 +28,12 @@ elif [[ -d /scratch3 ]] ; then
...
-    unset _LMFILES_
-    unset LOADEDMODULES
-    module use /opt/modulefiles
-    module use /opt/cray/ari/modulefiles
-    module use /opt/cray/craype/default/alt-modulefiles
-    module use /opt/cray/alt-modulefiles
-    module use /gpfs/hps/nco/ops/nwprod/modulefiles
-    module use /gpfs/hps/nco/ops/nwprod/lib/modulefiles
-    module use /usrx/local/prod/modulefiles
 elif [[ -d /dcom && -d /hwrf ]] ; then
```
Comment (BenjaminBlake-NOAA) [inline modulefiles/module-setup.sh.inc]: Recommend removing this section for Tide and Gyre.

**Review by BenjaminBlake-NOAA — CHANGES_REQUESTED** [review]
The changes look good, I'd just recommend removing the sections for Tide and Gyre which no longer exist.

File: modulefiles/module-setup.csh.inc (hunk 1 above)
Comment (chan-hoo) [reply modulefiles/module-setup.csh.inc]: @BenjaminBlake-NOAA, Removed.

File: modulefiles/module-setup.sh.inc (hunk 2 above)
Comment (chan-hoo) [reply modulefiles/module-setup.sh.inc]: Removed.

**Comment by chan-hoo** [comment]
@BenjaminBlake-NOAA, thank you for the comment. I've removed the blocks for Tide and Gyre from the if-statement.

**Review by BenjaminBlake-NOAA — APPROVED** [review]
@chan-hoo  Thanks for making the changes!

**Review by JeffBeck-NOAA — APPROVED** [review]

**Pull request merged**